app/main.py

```diff
@@ -16,4 +16,7 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [my_favourite_films, marks, collection_of_coins],
+    "immutable": [lucky_number, pi, one_is_a_prime_number, name, profile_info]
```

Comment:
The `profile_info` variable is a tuple, which is indeed immutable. However, the `name` variable is a string, which is also immutable in Python. The categorization of `name` as immutable is correct, but ensure that the understanding of immutability is clear for future reference.